tests/alembic_migrations/versions/v_78260d36e42b_add_net_var_name_valid_identifier_check/test_smoke.py

```diff
@@ -54,7 +54,7 @@ def test_upgrade(
     ]
 
     def inserter(x):
-        return get_insert_statement(schema_name, x[2], x[0])
+        return get_insert_statement(schema_name, x[1], x[0])
```

Comment:
Note: This wasn't directly in the scope of the issue but noticed that I had used the incorrect tuple entry while adding the current migration.